app/models/apple_credential.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AppleCredential < ActiveRecord::Base
+  belongs_to :public_feed, -> { with_deleted }, class_name: "Feed"
+  belongs_to :private_feed, -> { with_deleted }, class_name: "Feed"
```

Comment:
I'm realizing that `Feed`s don't have a `deleted_at` column.  So this `-> { with_deleted }` scope can be removed.  I'm actually surprised that doesn't throw an error?  `Feed.with_deleted.count` certainly does.

Reply:
Can that syntax be deleted without any other changes? I can take it out and re-commit.

Reply:
Yep - and hopefully the tests would break if the syntax was invalid.

Reply:
Ran the file individually and got no errors